pyproject.toml

```diff
@@ -44,7 +43,25 @@ select = [
     "D",
 ]
 ignore = [
-    "D105", "D203", "D213", "E501"
+    "D105",
+    "D203",
+    "D213",
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    "W191",
+    "E111",
+    "E114",
+    "E117",
+    "D206",
+    "D300",
+    "Q000",
+    "Q001",
+    "Q002",
+    "Q003",
+    "COM812",
+    "COM819",
+    "ISC001",
+    "ISC002",
```

Comment:
Do we need those ignores? We don't use those rules; they are not included as part of #230 and are not enabled by default. I don't see a reason for ignoring the rules that are not enabled.